topic3/src/keyPoint1/AmountConverterTester.java

```diff
@@ -0,0 +1,124 @@
+package keyPoint1;
+
+import org.junit.Test;
+
+public class AmountConverterTester {
+
+	@Test
+	public void testSplitAmount() {
+		AmountConverter amountConverter = new AmountConverter("23.34");
+		System.out.println(amountConverter.getDigits() + " " + amountConverter.getCents());
```

Comment:
Hello, I have corrected it all, now it is with Assert methods in all the key points.